# Parameters for control loop
pub_twist_cmd: true
debug_info: false
delay_mode: false
max_speed: 1.2 # unit: m/s 
waypoints_dist: -1.0 # unit: m, set < 0 means computed by node
goal_radius: 0.21 # unit: m
controller_freq: 30

# Parameter for MPC solver
mpc_steps: 20.0
mpc_ref_cte: 0.0
mpc_ref_etheta: 0.0
mpc_w_cte: 1000.0
mpc_w_etheta: 1000.0
mpc_w_vel: 2000.0
mpc_w_angvel: 0.0
mpc_w_angvel_d: 20.0
mpc_w_accel: 10.0
mpc_w_accel_d: 10.0
mpc_max_angvel: 2.0
mpc_max_throttle: 0.5 # Maximal throttle accel
mpc_bound_value: 1.0e3 # Bound value for other variables



